site/src/components/Resources/AgentRow.tsx

```diff
@@ -8,7 +8,7 @@ import {
   OpenDropdown,
```

Comment:
I think I'm lost... where is the "port forward" button?

<img width="1510" alt="Screenshot 2023-08-03 at 18 15 15" src="https://github.com/coder/coder/assets/14044910/f9560884-608d-4609-b412-96ca5c7a696a">

Reply:
You only will see it if it is set up in your workspace/template.

Reply:
Could you add the part of the template that enables it to the PR description?

Reply:
Let me share with you the docs: https://coder.com/docs/v2/latest/networking/port-forwarding#dashboard